tests/sigaltstack_test.c

```diff
@@ -178,7 +183,20 @@ GREATEST_MAIN_DEFS();
 
 int main(int argc, char** argv)
 {
-   printf("=== %d %d\n", SIGSTKSZ, MINSIGSTKSZ);
+   /*
+    * musl does not support MINSIGSTKSZ and SIGSTKSZ
+    * when compiled with _GNU_SOURCE
+    */
+   min_signal_stack_size = MINSIGSTKSZ;
+   signal_stack_size = SIGSTKSZ;
+
+   if (min_signal_stack_size <= 0) {
+      min_signal_stack_size = getauxval(AT_MINSIGSTKSZ);
```

Comment:
You should fail the test if the call to getauxval() fails.

We got to this point because we weren't validating what SIGSTKSZ and MINSIGSTKSZ were returning.

Reply:
added assertion